app/validators/fee/concerns/case_numbers_validator.rb

```diff
@@ -8,7 +8,9 @@
 module Fee
   module Concerns
     module CaseNumbersValidator
-      CASE_NUMBER_PATTERN = BaseValidator::CASE_NUMBER_PATTERN
+      URN_PATTERN = BaseValidator::URN_PATTERN
```

Comment:
**not-a-blocker* - i know you are merely extending what came before but seems better to me to either refer directly to constant in BaseValidator or define the constants here and refer to them directly where needed elsewhere.

Reply:
👍